TYPO3.Flow/Resources/Private/Installer/Distribution/Defaults/.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.php]
+indent_type = space
+indent_size = 4
+
+[*.yaml]
+indent_type = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Why is this a good idea and/or needed?

Reply:
Because double whitespace at the end of a line in markdown enforces a line break. http://markdown-guide.readthedocs.io/en/latest/basics.html#line-return